packages/amplify_api/lib/method_channel_api.dart

```diff
@@ -155,31 +153,34 @@ class AmplifyAPIMethodChannel extends AmplifyAPI {
       _subscription.cancel();
     });
 
-    Function cancel = () {
+    void cancel() {
       _subscription.cancel();
       cancelRequest(subscriptionId);
-    };
+    }
 
     return GraphQLSubscriptionOperation(cancel: cancel);
   }
 
   Future<GraphQLResponse<T>> _getMethodChannelResponse<T>({
-    required methodName,
+    required String methodName,
     required GraphQLRequest<T> request,
   }) async {
     try {
-      final Map<String, dynamic>? result =
-          (await (_channel.invokeMapMethod<String, dynamic>(
+      final result = await _channel.invokeMapMethod<String, dynamic>(
```

Comment:
Is the removal of explicit type annotations part of the linting?

Reply:
Yep: https://dart-lang.github.io/linter/lints/omit_local_variable_types.html